### PR TITLE
Fix/Capitalize all descriptive section titles and buttons

### PR DESCRIPTION
**Pull request by erin-koen**
Capitalized all words in the vast majority of section titles and all buttons. The only exceptions were sectiontitles nested within liquidityProviderTrading and RFQTrading components, as these are less descriptive and more instructive and all caps looked like we were yelling at the user.

### CODE REVIEW

**Comment by erin-koen** [comment]
Also, this touches basically every file so likely to cause merge conflicts . Maybe best course of action is to wait till end of day in europe to merge then people can pull a fresh master tomorrow morning?

File: src/components/Routes/Fund/FundTrade/FundLiquidityProviderTrading/FundLiquidityProviderTrading.tsx [hunk 1]
```diff
@@ -103,13 +103,13 @@ export const FundLiquidityProviderTrading: React.FC<FundLiquidityProviderTrading
...
     <Block>
-      <SectionTitle>Liquidity pools trading</SectionTitle>
+      <SectionTitle>Liquidity Pool Trading</SectionTitle>
 
       <FormContext {...form}>
         <Grid>
           <GridRow>
             <GridCol>
-              <SectionTitle>Choose the assets to swap</SectionTitle>
+              <SectionTitle>Choose the assets to swap:</SectionTitle>
```
Comment (fubhy) [inline src/components/Routes/Fund/FundTrade/FundLiquidityProviderTrading/FundLiquidityProviderTrading.tsx]: I think the colons in the titles look weird. At least on windows they look really out ouf place.

File: src/components/Routes/Wallet/WalletWeth/WalletWeth.tsx [hunk 2]
```diff
@@ -5,13 +5,15 @@ import WalletUnwrapEther from './WalletUnwrapEther/WalletUnwrapEther';
 
 import { Grid, GridRow, GridCol } from '~/storybook/components/Grid/Grid';
 import { NotificationBar, NotificationContent } from '~/storybook/components/NotificationBar/NotificationBar';
+import { Link } from '~/storybook/components/Link/Link';
```
Comment (fubhy) [inline src/components/Routes/Wallet/WalletWeth/WalletWeth.tsx]: New import not needed.

File: src/components/Routes/Fund/FundTrade/FundLiquidityProviderTrading/FundLiquidityProviderTrading.tsx (hunk 1 above)
Comment (erin-koen) [reply src/components/Routes/Fund/FundTrade/FundLiquidityProviderTrading/FundLiquidityProviderTrading.tsx]: fair enough - i was on the fence. Took them out.

**Pull request merged**